src/locals/pt-br.json

```
@@ -0,0 +1,141 @@
+{
+  "onboarding_screen": {
+    "auto_thought_ex": "Me esqueci da festa do Jorge! Ele deve me odiar.",
+    "challenge_ex": "O Jorge sabe que não consigo ir a todas as festas. Eu não o odiaria caso se esquecesse da minha. Tenho que me considerar tão bem quanto considero os outros.",
+    "alt_thought_ex": "É verdade que me esqueci da festa do Jorge mas posso pedir desculpa e ele compreenderá.",
+    "cog_distortion": { "label": "Leitura da mente", "slug": "leitura-mente" },
+    "intro": {
+      "line1": "Bem vindo ao",
+      "emphasis1": "quirk",
+      "line2": ", uma",
+      "emphasis2": "app",
+      "line3": " de terapia cognitivo comportamental (TCC)."
+    },
+    "crash_course": {
+      "line1": "Isto é um",
+      "emphasis1": "curso rápido",
+      "line2": "de TCC e ",
+      "emphasis2": "não uma alternativa",
+      "line3": "a um terapeuta."
+    },
+    "mood_thoughts": {
+      "line1": "Primeiro:",
+      "emphasis1": "Os seus pensamentos causam as suas emoções."
+    },
+    "thoughts_arent_thoughts": {
+      "line1": "Segundo: Os seus pensamentos podem ser",
+      "emphasis1": "automáticos."
+    },
+    "your_distorted": {
+      "line1": "Terceiro: Pensamentos automáticos estão frequentemente",
+      "emphasis1": "distorcidos."
+    },
+    "cbt_overview": {
+      "line1": "O objetivo de TCC é reconhecer os",
+      "emphasis1": "pensamentos distorcidos",
+      "line2": "e",
+      "emphasis2": "substituí-los",
+      "line3": "por pensamentos racionais."
+    },
+    "ready": {
+      "emphasis1": "Pronto?",
+      "line1": "Vamos passo-a-passo."
+    },
+    "catch": {
+      "line1": "1. Vamos começar por",
+      "emphasis1": "escrever",
+      "line2": "os pensamentos automáticos assim que acontecerem."
+    },
+    "bad_thought_note": {
+      "line1": "Repare o pensamento:",
+      "line2": "Me esqueci da festa do Jorge. Ele deve me odiar."
+    },
+    "identify_distortions": {
+      "line1": "Procure por",
+      "emphasis1": "distorções cognitivas"
+    },
+    "bad_distortions": {
+      "line1": "Sabemos mesmo se o Jorge vai nos odiar? Se não, estamos fazendo",
+      "emphasis1": "leitura da mente.",
+      "line2": "Me esqueci da festa do Jorge. Ele deve me odiar."
+    },
+    "challenge_screen": {
+      "line1": "3. Porque é que este pensamento não é realista?",
```

Would this be better? 
```suggestion
      "line1": "3. Por que este pensamento não é realista?",
```